### ISSUE TITLE
How to resume from stage-1 pretrained model

### ISSUE THREAD

**Issue by htwang14** [kminsoo/AFC#4]
Hi, 

Thanks for sharing the codes of your great work!

I suppose the stage 1 model trained on the 50 base classes should be reusable for different incremental settings (e.g., `--increment 1`, `--increment 10`, etc.).

So, can I use the following command to reuse the base classifier?

```
python3 -minclearn --options options/AFC/AFC_cnn_cifar100.yaml options/data/cifar100_3orders.yaml \
    --initial-increment 50 --increment 1 --fixed-memory \
    --device <GPU_ID> --label AFC_cnn_cifar100_50steps \
    --data-path <PATH/TO/DATA>\
--resume <WHERE/I/STORED/THE/PTH/FILES> --resume_first
```

The `<WHERE/I/STORED/THE/PTH/FILES>` folder contains files `net_<run_ID>_task_0.pth`.

Also, I think [this resume condition](https://github.com/kminsoo/AFC/blob/94cecc9107479c628a7d11690a7419edb7a41bf0/inclearn/train.py#L214) is not functioning as expected. Should it be replaced by the following:

```
model.load_parameters(config["resume"], run_id)
logger.info(
            "Skipping initial training phase {} because reloading pretrained model.".
            format(task_id)
        )
if config["model"] == "afc":
            model._update_importance(train_loader)
```

If this fix is made, resuming from base classifier would be easier by the following command:

```
python3 -minclearn --options options/AFC/AFC_cnn_cifar100.yaml options/data/cifar100_3orders.yaml \
    --initial-increment 50 --increment 1 --fixed-memory \
    --device <GPU_ID> --label AFC_cnn_cifar100_50steps \
    --data-path <PATH/TO/DATA>\
--resume <WHERE/I/STORED/THE/PTH/FILES/net_<run_ID>_task_0.pth>
```

I would appreciate it if you could confirm this. And please correct me if I'm wrong. 

Thank you in advance!

**Comment by htwang14** [comment]
BTW, I think [this line](https://github.com/kminsoo/AFC/blob/94cecc9107479c628a7d11690a7419edb7a41bf0/inclearn/lib/results_utils.py#L33) should be changed to `file_path = "run_{}_.json".format(run_id)`. Otherwise the results on previous class orders would be overwritten.

**Comment by kminsoo** [comment]
Hi, thank you for your interest.

**Comment by kminsoo** [comment]
Hi, htwang14.
Thank you for your interest.
If you want to resume the first stage, you should use the following command as you mentioned:

    python3 -minclearn --options options/AFC/AFC_cnn_cifar100.yaml options/data/cifar100_3orders.yaml \
    --initial-increment 50 --increment 1 --fixed-memory \
    --device <GPU_ID> --label AFC_cnn_cifar100_50steps \
    --data-path <PATH/TO/DATA>\ 
    --resume <WHERE/I/STORED/THE/PTH/FILES> --resume_first

For example, <WHERE/I/STORED/THE/PTH/FILES> is not

    ./results/dev/afc/202208/week_5/20220829_AFC_cnn_cifar100_25steps/net_<run_id>_task_0.pth

but

    ./results/dev/afc/202208/week_5/20220829_AFC_cnn_cifar100_25steps



Based on the command, I have just found that it works well.
If you still find it does not work, please feel free to ask about your problem!

**Comment by htwang14** [comment]
Thank you!
